homeassistant/components/lamarzocco/water_heater.py

```diff
@@ -0,0 +1,162 @@
+"""Water heater platform for La Marzocco espresso machines."""
```

Comment:
A water heater platform is supposed to be used for boiler or house water heating systems. Please refactor it

Reply:
I'm using it to represent the two water boilers of the coffee machine (coffee & steam) so in that sense I think that fits. Don't you agree?

Reply:
I will check this with the team and come back

Reply:
We came internally to the conclusion that water_heater should not used for a coffee machine. You can expose the information which number and sensor entities. Please adopt the code.

Please note that you should only add one platform at the time in one PR

Reply:
Need to think about that, might just abandon this PR then...

Reply:
Trying to understand the decision. What confuses me the most: Even if it was intended for something different why can't I use it here? Replicating 98% of the code into a separate platform just for the sake of intended use really doesn't make any sense to me... If I were to switch to number and sensor entities the information is all over the place (especially when translating to HomeKit, which I and a lot of my users do as far as I've heard, compared to a HomeKit thermostat)